api/src/evaluation/domain/models/AutonomousCourse.js

```diff
@@ -0,0 +1,11 @@
+export class AutonomousCourse {
+  constructor({ id, internalTitle, publicTitle, customLandingPageText, createdAt, code }) {
+    this.id = id;
+    this.internalTitle = internalTitle;
+    this.publicTitle = publicTitle;
+    this.customLandingPageText = customLandingPageText;
+    this.publicTitle = publicTitle;
```

Comment:
```suggestion
    this.publicTitle = publicTitle;
    this.customLandingPageText = customLandingPageText;
```

Il y a un `this.publicTitle = publicTitle;` en trop